documentation/modules/ROOT/pages/connectors/jdbc.adoc

```diff
@@ -700,6 +1165,7 @@ Information about the properties is organized as follows:
 * xref:jdbc-connector-properties-extendable[JDBC connector extendable properties]
 * xref:jdbc-connector-hibernate-passthrough-properties[JDBC connector `hibernate.*` passthrough properties]
 
+// Type: reference
 [[jdbc-connector-properties-generic]]
 === JDBC connector generic properties
```

Comment:
I wonder if this section should be renamed to something like "Kafka Connect consumer properties"?

Using the naming "JDBC connector generic properties" seems to indicate to me that these are specific to the JDBC sink when in reality, these aren't used by the JDBC sink at all but rather the Kafka Connect runtime.  In a similar way, the same holds true to the newly added `class` property, although for the downstream it's Strimzi specific, which passes it onto Kafka Connect.

Reply:
@Naros TBH, I didn't take a further look at content beyond the deployment steps. 
I agree about changing the title of this section. WRT the `class` property, would you say that we should add that -- a least for the downstream docs -- to the properties listed in this section?